func.md

```diff
@@ -0,0 +1,86 @@
+# Functionalities
+
+## autothread
+This functionality creates automatic threads on every new message in a channel. Can be helpful in channels dedicated to questions. <br>
```

Comment:
Instead of `<br>` you can add two spaces after the line to make a line break. However, it's not necessary most of the time (e.g. here, for example).

Reply:
I left most of the newlines for the purpose of formatting but switched them all to doublespace. Waiting for your feedback again.